.github/pull_request_template.md

```diff
@@ -1,23 +1,17 @@
-> Add a list of changes
```

Comment:
A description of the changes should be first since this is all you see in notifications. We have the PR title above.